src/hdmf_zarr/backend.py

```diff
@@ -1229,3 +1226,16 @@ def __read_attrs(self, zarr_obj):
                 else:
                     ret[k] = v
         return ret
+
+    @docval({'name': 'zarr_obj', 'type': (Array, Group),
+             'doc': 'the Zarr object to the corresponding Container/Data object for'})
+    def get_container(self, **kwargs):
```

Comment:
Very minor. I would move the function to make sure `get_container` and `get_builder` are in the same place in the code, since the two are related.